_posts/2019-08-06-serverless-static-sites.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Bringing Serverless to a webpage near you with Hugo & Kubernetes"
+description: With the simplicity of OpenFaaS's templating system we can easily create new templates in no time. Today we will be showing how to continuosly deploy static sites using a new template for hugo and the FunctionIngress CRD for custom domain names.
```

Comment:
"we will be showing how to " -> this can be simplified with fewer words.

Reply:
"we'll show how to" maybe?